src/incident_reports/configs/init_mongo_prod.js

```diff
@@ -0,0 +1,10 @@
+db = db.getSiblingDB('test_concierge');
```

Comment:
Multi-database management shouldn't be necessary since this entire server will only ever handle this single db